core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -112,6 +112,53 @@ class TransactionsTest extends KafkaServerTestHarness {
   }
 
   @Test
+  def testReadCommittedConsumerShouldNotSeeUndecidedData(): Unit = {
```

Comment:
From a terminology perspective, what is the difference between uncommitted and undecided data?